app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -213,7 +213,7 @@ class BrowserFragment :
             PromptFeature(
                 fragment = this,
                 store = components.store,
-                customTabId = tab.id,
+                customTabId = tryGetCustomTabId(),
```

Comment:
We were passing the tab `id` instead of the `customTabId`, causing `PromptFeature` to get confused [here](https://github.com/mozilla-mobile/android-components/blob/c41a8e569b83ac1c6b48a673aadfca522cab3e13/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/file/FilePicker.kt#L137)